app/src/main/java/org/systers/mentorship/view/activities/AboutActivity.kt

```diff
@@ -0,0 +1,65 @@
+package org.systers.mentorship.view.activities
+
+import android.os.Bundle
+import kotlinx.android.synthetic.main.activity_about.*
+import org.systers.mentorship.BuildConfig
+import org.systers.mentorship.R
+import android.content.Intent
+import android.net.Uri
+import org.systers.mentorship.R.string.username
+
+
+
+/**
+ * This activity displays information about this App
+ */
+class AboutActivity : BaseActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_about)
+
+        supportActionBar?.title = getString(R.string.title_activity_about)
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+        supportActionBar?.setDisplayShowHomeEnabled(true)
+        val res = resources
+        val version = String.format(res.getString(R.string.version_string), BuildConfig.VERSION_CODE)
```

Comment:
I actually meant something like "Version: ${BuildConfig.VERSION_CODE}" but this is better, nice work!

Reply:
Thanks !